siliconcompiler/report/dashboard/cli/board.py

```diff
@@ -282,14 +291,13 @@ def _init_singleton(self):
     def open_dashboard(self):
         """Starts the dashboard rendering thread if it is not already running."""
 
-        if not self._active:
-            return
```

Comment:
this test is needed when running in a non-tty terminal. You can test this by setting your environmental variable to TTY_COMPATIBLE=0

Reply:
I'll put it back.